project.clj

```diff
@@ -20,7 +20,7 @@
   ;
   :aliases { "examples" ["run" "-m" "seesaw.test.examples.launcher"] }
 
-  :dependencies [[org.clojure/clojure "1.8.0"]
+  :dependencies [[org.clojure/clojure "1.4.0"]
```

Comment:
Why is that?

Reply:
Reverted to 1.8.0 again

Reply:
Just tried to find out how jenkins tests work

Reply:
Ah ok